src/lib.rs

```diff
@@ -53,7 +103,7 @@ impl Builder {
     }
 
     pub fn include_google_protos(&mut self) -> &mut Self {
-        let path = format!("{}/include", std::env!("CARGO_MANIFEST_DIR"));
+        let path = format!("{}/include", env!("CARGO_MANIFEST_DIR"));
```

Comment:
Maybe protos in include directory also need to be updated.

Reply:
Updated.